p2p/key.go

```diff
@@ -65,26 +76,23 @@ func LoadNodeKey(filePath string) (*NodeKey, error) {
 	nodeKey := new(NodeKey)
 	err = cdc.UnmarshalJSON(jsonBytes, nodeKey)
 	if err != nil {
-		return nil, fmt.Errorf("error reading NodeKey from %v: %v", filePath, err)
+		return nil, err
 	}
 	return nodeKey, nil
 }
 
-func genNodeKey(filePath string) (*NodeKey, error) {
-	privKey := ed25519.GenPrivKey()
-	nodeKey := &NodeKey{
-		PrivKey: privKey,
-	}
-
-	jsonBytes, err := cdc.MarshalJSON(nodeKey)
+// SaveNodeKey persists the given NodeKey to filePath.
+func SaveNodeKey(key *NodeKey, filePath string) error {
+	jsonBytes, err := cdc.MarshalJSON(key)
 	if err != nil {
-		return nil, err
+		return err
 	}
-	err = ioutil.WriteFile(filePath, jsonBytes, 0600)
+	const perm = 0600
```

Comment:
Why the `const`?

Reply:
so it does not change?

Reply:
But why not pass 0600 directly to the function?